host/sgx/enter.c

```diff
@@ -8,8 +8,11 @@
 #include <openenclave/internal/sgx/ecall_context.h>
 #include "asmdefs.h"
 #include "enclave.h"
+#include "vdso.h"
 #include "xstate.h"
 
+extern bool oe_sgx_is_vdso_enabled;
```

Comment:
To prevent redeclaration with incorrect type, it is best to declare this in a header file and define it in a C file.